uitools/import/Esri/ArcGISRuntime/Toolkit/PopupView.qml

```diff
@@ -190,7 +193,10 @@ Pane {
                 Label {
                     Layout.fillWidth: true
                     text: formattedValue
-                    wrapMode: Text.WrapAnywhere
+                    wrapMode: Text.Wrap
+                    palette: popupView.palette
+                    font: popupView.font
+                    horizontalAlignment: Text.AlignRight
```

Comment:
💭 I think I preferred with a left align, it is a bit easier to read and it is more table-like

Reply:
Very fair, I have reverted back!